src/wordy_webserver_response.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+class WebServerResponse {
+  constructor(storage) {
+    this.storage = storage;
+  }
+
+  static getHome() {
+    return 'WordyBot';
+  }
+
+  processCommand(req, callback) {
+    const command = req.body.command;
+    const userId = req.body.user_id;
+
+    console.log('Command received');
+    console.log(`Command: ${command}`);
+    console.log(`User ID: ${userId}`);
+
+    switch (command) {
```

Comment:
A bit a matter of taste, a bit how brittle is a `switch` (e.g. forget a `break`). I'd go with something like:

```js
const commands = {
  '/wordy-in': this.userIn,
  '/wordy-out': this.userOut,
};
const isValidCommand = Object.keys(commands).some(x => command == x);

if (!isValidCommand) {
  console.warn('Unknown command, ignoring');
  return callback('Unknown command.');
}

return commands[command](userId, callback);
```